WordPressAuthenticator/Signin/LoginSiteAddressViewController.swift

```diff
@@ -226,7 +226,7 @@ class LoginSiteAddressViewController: LoginViewController, NUXKeyboardResponder
     }
 
     func presentNextControllerIfPossible(siteInfo: WordPressComSiteInfo?) {
-        WordPressAuthenticator.shared.delegate?.shouldPresentUsernamePasswordController(for: siteInfo, onCompletion: { (error, isSelfHosted) in
+        WordPressAuthenticator.shared.delegate?.shouldPresentUsernamePasswordController(for: siteInfo, onCompletion: { (error, isSelfHosted, injectedViewController) in
             guard let originalError = error else {
```

Comment:
`LoginSiteAddressViewController` is the old auth flow (i.e. it will be nixed in the future). This should be done in`SiteAddressViewController`, which is the new, unified Site Address VC.